cylc/flow/scripts/install.py

```diff
@@ -297,19 +296,12 @@ def install(
     # for compatibility mode:
     check_deprecation(source)
 
-    for entry_point in iter_entry_points(
-        'cylc.pre_configure'
+    async for _entry_point, _plugin_result in run_plugins_async(
```

Comment:
Why is this async, since pre-config doesn't need it?

Reply:
Why *should* it be sync?